redi-connect-front/src/styles/_variables.scss

```diff
@@ -40,7 +41,7 @@ $redi-blue-light: #deeff3;
 
 $redi-orange-dark: #ea5b29;
 $redi-orange: #f6b9a2;
-$redi-orange-light: #f6b9a2;
+$redi-orange-light: #FAD7CA;
```

Comment:
This is incorrect. Please revert to the value which was there. 
https://deloittedigitalde.invisionapp.com/d/main#/console/19625674/410340231/preview